crates/e2e-tests/README.md

```diff
@@ -0,0 +1,57 @@
+# TIPS E2E Tests
+
+End-to-end integration tests for the TIPS (Transaction Inclusion Protocol Service) system.
+
+## Prerequisites
+
+All tests require the full infrastructure from `SETUP.md` running:
+- TIPS ingress service (port 8080) via `just start-all`
+- builder-playground (L1/L2 blockchain) on `danyal/base-overlay` branch
+- op-rbuilder (block builder)
+- Kafka
+- MinIO
+
+## Running Tests
+
+### Start Infrastructure
+
+Follow guidelines in `SETUP.md`
+
+### Run Integration Tests
+
+```bash
+cd tips
+INTEGRATION_TESTS=1 cargo test --package tips-e2e-tests -- --nocapture
```

Comment:
qq to confirm: if i make a change to the implementation (not the test), i'll still need to re-run `just start-all` right?